pkg/image/inspector.go

```
@@ -91,19 +93,32 @@ func (i *registryInspector) GetCompatibleArchitecturesSet(ctx context.Context, i
 	// than do this everytime
 	sysregistriesv2.InvalidateCache()
 
-	// Check if the image is a manifest list
-	ref, err := docker.ParseReference(imageReference)
-	if err != nil {
-		log.Error(err, "Error parsing the image reference for the image")
-		return nil, err
-	}
 	sys := &types.SystemContext{
 		AuthFilePath:                authFile.Name(),
+		RegistriesDirPath:           RegistryCertsDir(),
 		SystemRegistriesConfPath:    RegistriesConfPath(),
-		SystemRegistriesConfDirPath: RegistryCertsDir(),
+		SystemRegistriesConfDirPath: RegistriesConfDir(),
 		SignaturePolicyPath:         PolicyConfPath(),
 		DockerPerHostCertDirPath:    DockerCertsDir(),
 	}
+	// we expect the imageReference to start with `//`. Let's remove it for shortnames func
+	if strings.HasPrefix(imageReference, "//") {
```

If we expect the imageReference to always start with `//`, can we skip the if statement and just calculate the trimmedRef? 

At most, if we want to double check the imageReference starts with `//`, i'd do something like:

```
if !strings.HasPrefix(imageReference, "//") {
  return nil, errors.New("invalid value for the imageReference argument")
}
```

In the current version, if the imageReference does not start with `//`, we silently skip checking for shortnames.